generated/nifake/nifake_library.cpp

```diff
@@ -378,11 +391,7 @@ ViStatus NiFakeLibrary::GetError(ViSession vi, ViStatus* errorCode, ViInt32 buff
   if (!function_pointers_.GetError) {
     throw nidevice_grpc::LibraryLoadException("Could not find niFake_GetError.");
   }
-#if defined(_MSC_VER)
```

Comment:
Yes, I was shocked at first, but then I remember that being the case when this change was originally made. There was a discussion about what kind of metadata tag we should use for functions that were in a private header and not exposed in the headers we uploaded to this repo. The `private` tag made sense, though `nifake` was already using it quite a bit. Since we don't actually use the concrete `NIFakeLibrary` in any of our tests, we decided not to update the nifake metadata to remove the `private` tags off of those functions.